namerd/storage/k8s/src/main/scala/io/buoyant/namerd/storage/K8sDtabStore.scala

```diff
@@ -69,7 +72,7 @@ class K8sDtabStore(client: Http.Client, dst: String, namespace: String)
   // semantics. If those are required, we would probably need to implement a synchronized means of
   // reading `act` and using that value to feed to `witness`, rather than taking the `foldLeft`
   // approach as below.
-  private[this] val act: Activity[NsMap] =
+  private[this] val act: InstrumentedActivity[NsMap] =
     watchApi.dtabs.activity(dtabListToNsMap) {
```

Comment:
We'll also want to create a WatchState, pass it into activity, and serialize it the admin handler.